public/contribute.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "blurts-server",
+    "description": "The app powering monitor.firefox.com.",
+    "repository": {
+      "url": "https://github.com/mozilla/blurts-server",
+      "license": "MPL2",
```

Comment:
Not sure if anything validates these, but this should *maybe* be "MPL-2.0" to match package.json's SPDX license format.